### PR TITLE
Fix belongsTo

### PR DESCRIPTION
**Pull request by rybakk**

### CODE REVIEW

File: tests/IntegrationTest.php [hunk 1]
```diff
@@ -113,6 +113,60 @@ private function createRoleRaw()
         return $this;
     }
 
+    private function createCustomerRaw()
```
Comment (gcphost) [inline tests/IntegrationTest.php:116]: Double paste?

File: tests/IntegrationTest.php [hunk 2]
```diff
@@ -113,6 +113,60 @@ private function createRoleRaw()
...
+
+        $phone = new Phone(['number' => '1231231234']);
+        $phone->customer()->associate($customer);
+        $phone->room()->associate($room);
+        $phone->save();
+
+        return $customer;
+    }
+
+    public function testBelongsToRelation()
```
Comment (gcphost) [inline tests/IntegrationTest.php:129]: Here too

**Review by gcphost — CHANGES_REQUESTED** [review]

**Comment by maguilar92** [comment]
@gcphost I think there's no need to make package work with belongsTo relation.
The cascading must be defined in hasOne/hasMany, example:

Car hasMany Components
Component belongsTo Car
$component->car_id = $car->id
cascading will be defined in Car because component depends on car and not the opposite. If I delete a component the car should not be erased.

**Comment by gcphost** [comment]
Your call @maguilar92 on that.

This PR has a lot of `Immofacile` name space changes, I didn't see that before. It'll have to be rejected.

**Pull request closed without merging**